lib/MusicBrainz/Server/Data/Alias.pm

```diff
@@ -88,8 +88,8 @@ sub find_by_entity_ids
                    end_date_year NULLS LAST,
                    end_date_month NULLS LAST,
                    end_date_day NULLS LAST,
-                   musicbrainz_collate(sort_name),
-                   musicbrainz_collate(name)";
+                   sort_name COLLATE musicbrainz,
+                   name COLLATE musicbrainz";
```

Comment:
Unfortunately creating a function that returns `input COLLATE musicbrainz` didn't work, because that doesn't affect the expression outside the function call. So I didn't see a way to do this without breaking the code (short of defining the collation in the table definition, but changing the data type of a column isn't that easy).